lib/msal-browser/src/broker/nativeBroker/NativeRequest.ts

```diff
@@ -52,3 +52,30 @@ export type NativeExtensionRequest = {
     extensionId?: string;
     body: NativeExtensionRequestBody;
 };
+
+export type PlatformDOMTokenRequest = {
+    brokerId: string;
+    accountId?: string;
+    clientId: string;
+    authority: string;
+    scope: string;
+    redirectUri: string;
+    correlationId: string;
+    isSecurityTokenService: boolean;
```

Comment:
Is this something we would ever set?

Reply:
This is to identify if this is coming from STS and the naming is again generalized for standardization purposes.

Reply:
My point is that MSAL.js will never set this and thus doesn't need to be part of our code.